hil/cli.py

```diff
@@ -797,24 +797,13 @@ def create_admin_user(username, password):
 
 
 @cmd
-def list_active_extensions(ext_type):
-    """List active extensions by type.
-
-    Default types: auth, obm, switches, network_allocators.
-
-    Use the 'all' argument to list all active extensions.
-    """
+def list_active_extensions():
+    """List active extensions by type. """
     if not config.cfg.has_section('extensions'):
         sys.exit("There are no active extensions.")
```

Comment:
I modified this to check if the list from the API is empty rather than from the config.